source/install/self-managed-changelog.md

```diff
@@ -4,12 +4,130 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v6.5 - Feature Release](#release-v6-5-feature-release)
 - [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 - [Release v6.2 - Feature Release](#release-v6-2-feature-release)
-- [Release v6.1 - Feature Release](#release-v6-1-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5.37-extended-support-release)
 
+## Release v6.5 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2022-03-16**
+
+### Compatibility
+ - Updated the recommended minimum supported Firefox version to v91+.
+
+### Important Upgrade Notes
+ - The ``mattermost version`` CLI command has been deprecated and does not interact with the database anymore. Therefore the database version is not going to be printed. Also, the database migrations are not going to be applied with the version sub command. A new db migrate sub command is added to enable administrators to trigger migrations.
```

Comment:
Here is a link to the mattermost db migrate documentation in case you'd like to add a direct link: https://docs.mattermost.com/manage/command-line-tools.html#mattermost-db-migrate